### DISCUSSION TITLE
Upgraded to latest teslamate, teslamate/grafana, postres:16, mosquitto:2 no - Port 4000 not responding

### DISCUSSION THREAD

**Discussion by jweichel** [teslamate-org/teslamate#4060] [Getting started]
Docker PS:
pi@raspberrypi:~/Docker/Teslamate$ docker ps
CONTAINER ID   IMAGE                        COMMAND                  CREATED        STATUS                                  PORTS                                                 NAMES
6031b858977c   eclipse-mosquitto:2          "/docker-entrypoint.…"   17 hours ago   Up 13 seconds                           0.0.0.0:1883->1883/tcp, :::1883->1883/tcp             teslamate-mosquitto-1
6d4858c3bc8b   teslamate/grafana:latest     "/run.sh"                17 hours ago   Up 13 seconds                           0.0.0.0:3000->3000/tcp, :::3000->3000/tcp             teslamate-grafana-1
7c03468670d6   teslamate/teslamate:latest   "tini -- /bin/sh /en…"   17 hours ago   Up 3 seconds                            0.0.0.0:4000->4000/tcp, :::4000->4000/tcp             teslamate-teslamate-1
1c5361a5abb0   postgres:16                  "docker-entrypoint.s…"   17 hours ago   Restarting (1) Less than a second ago                                                         teslamate-database-1
16eb363c9e92   portainer/portainer-ce       "/portainer -H unix:…"   3 years ago    Up 18 hours                             8000/tcp, 0.0.0.0:9000->9000/tcp, :::9000->9000/tcp   management_portainer_1

Yet, if I look at the Container list on port 9000, I do not see port 4000 published
![ContainerList](https://github.com/teslamate-org/teslamate/assets/10646569/0c296383-48ec-4c14-b437-af92803f64d8)

**Comment by parkr** [comment]
Seems like a UI bug because the `docker ps` output seems right.

Feel free to drop your redacted docker-compose.yaml too if you think it could be misconfigured.

**Reply by jweichel** [reply]
[docker-compose.pdf](https://github.com/user-attachments/files/16135821/docker-compose.pdf)
I also noticed that the docker compose ps shows the service is continually restarting and eventually restarts but shows no ports.  See:
[Docker ps](https://github.com/teslamate-org/teslamate/assets/10646569/0b74eea1-126b-43c2-8582-0da6ba9aa0d5)

It looks like there is something wrong with teslamate.

**Reply by jweichel** [reply]
Should I have removed the previous teslamate DB since I upgraded the postgres?  Or maybe that is why the names teslamate-(grafana database, mosquito, teslamate) all have a -1 at the end.

**Reply by mariogolf** [reply]
Same problem. New debian installation and new docker installation. Only teslamate no listens on port 4000. Waiting for suggestions.

**Reply by mariogolf** [reply]
I've just fixed my issue. Problem was the docker folder's name ("teslamate"): I renamed it with "tm" and everything goes.

**Comment by jweichel** [comment]
I finally had to re-image the Raspberry Pi and install docker and teslamate.  It came up with port 4000 and worked well using a tesla token to login.
